documentation/cubic-pos-article.md

```diff
@@ -0,0 +1,155 @@
+# Introduction
+
+Proof-of-stake (PoS) consensus was developed as a way to solve the large computational overhead and long block finality times of proof-of-work (PoW) consensus mechanisms (e.g. in Bitcoin). Rather than miners competing to create the next block by hashing with lots of computing power in PoW systems, the block proposer is selected from a set of validators with probability proportional to the validator's bonded stake (or voting power). This block proposer mechanism allows PoS chains to be much greener than PoW chains that need to consume massive amounts of energy to reach consensus.
+
+Namada's PoS module is similar to and influenced by the PoS system of the Cosmos Hub. Like many other blockchains in the Cosmos ecosystem, Namada employs Tendermint as its PoS consensus engine. Tendermint is a byzantine fault-tolerant (BFT) system that provides deterministic state machine replication as long as at least $2/3$ of the consensus voting power agrees on the state of the machine. Voting power in the consensus mechanism is held by nodes called validators, and the voting power is proportional to the amount of staked tokens held by the validators.
+
+- Anything else?
+
+This article is divided into three main sections. First, we describe the mechanics and configuration of the epoched PoS system. Then we describe Namada's novel approach to slashing for validator misbehavior. Finally, we describe the inflation and rewards system within the PoS module.
+
+
+# Epoched Proof of Stake
+
+The native staking token in Namada is identified as *NAM*. We call the smallest denomination of this token *namnam*, where 1 million *namnam* is equal to 1 *NAM*.
+
+Namada's epoched PoS model is influenced by that of [Cosmos](https://github.com/cosmos/cosmos-sdk/blob/019444ae4328beaca32f2f8416ee5edbac2ef30b/docs/architecture/adr-039-epoched-staking.md#pipelining-the-epochs). Sets of blocks are batched into epochs, which are periods of time that have some fixed configuration of the PoS system. For example, the set of validators who participate in the consensus protocol and their associated consensus voting powers can only change at the boundary of two epochs and are constant throughout the duration of one.
+
+Namada employs epoched staking for two main reasons:
+- Light clients become more efficient because there are fewer state changes (particularly validator set changes) to store, resulting in fewer headers needed to verify state
```

Comment:
primarily because validator set changes are less frequent, so light clients don't need to download as many of them, and bisection becomes more efficient